src/person/person.spec.ts

```diff
@@ -21,6 +21,10 @@ chai.use(require('chai-http'));
 
 const dbIdExample = ['5b50a76713ddf90af494de32', '5b56e5ca07f0de0f38110b9c', '5b50a76713ddf90af494de33', '5b50a76713ddf90af494de34','5b50a76713ddf90af494de35','5b50a76713ddf90af494de36', '5b50a76713ddf90af494de37'];
```

Comment:
Delete dbIdExample whether not used

Reply:
It is used by the "responsibilityLocation" field. apparently, we do not check if this field refers to an existing group...